app/repositories/ReactiveRepositoryHelpers.scala

```diff
@@ -62,7 +62,7 @@ trait ReactiveRepositoryHelpers {
     }
   }
 
-  private[this] def singleUpdateValidatorImpl(id: String, actionDesc: String, ignoreNotFound: Boolean = false,
+  private[this] def singleUpdateValidatorImpl(id: String, actionDesc: String, ignoreNotFound: Boolean,
```

Comment:
The new compiler flags said it was unused (the public methods that call this private method all have their own default  values that hide this one)